test/modules/core/controllers/controllers.spec.js

```diff
@@ -15,7 +15,8 @@ test('MapController', async t => {
     latitude: 37.78,
     zoom: 10,
     pitch: 30,
-    bearing: -45
+    bearing: -45,
+    position: [0, 0, 0]
```

Comment:
If you have to add this for tests to pass then this will break every existing application that does not explicitly set `position` in its view state.

Reply:
Done